res/css/views/elements/_EditableItemList.scss

```diff
@@ -16,47 +16,38 @@ limitations under the License.
 
 .mx_EditableItemList {
     margin-top: 12px;
-    margin-bottom: 0px;
+    margin-bottom: 10px;
 }
 
 .mx_EditableItem {
-    display: flex;
-    margin-left: 56px;
+    margin-bottom: 5px;
+    margin-left: 15px;
 }
 
-.mx_EditableItem .mx_EditableItem_editable {
-    border: 0px;
-    border-bottom: 1px solid $strong-input-border-color;
-    padding: 0px;
-    min-width: 240px;
-    max-width: 400px;
-    margin-bottom: 16px;
+.mx_EditableItem_delete {
+    margin-right: 5px;
+    cursor: pointer;
+    vertical-align: middle;
 }
 
-.mx_EditableItem .mx_EditableItem_editable:focus {
-    border-bottom: 1px solid $accent-color;
-    outline: none;
-    box-shadow: none;
+.mx_EditableItem_email {
+    vertical-align: middle;
 }
 
-.mx_EditableItem .mx_EditableItem_editablePlaceholder {
-    color: $settings-grey-fg-color;
+.mx_EditableItem_promptText {
+    margin-right: 10px;
 }
 
-.mx_EditableItem .mx_EditableItem_addButton,
-.mx_EditableItem .mx_EditableItem_removeButton {
-    padding-left: 0.5em;
-    position: relative;
-    cursor: pointer;
-
-    visibility: hidden;
+.mx_EditableItem_confirmBtn {
+    margin-right: 5px;
 }
 
-.mx_EditableItem:hover .mx_EditableItem_addButton,
-.mx_EditableItem:hover .mx_EditableItem_removeButton {
-    visibility: visible;
+.mx_EditableItemList_newItem .mx_Field input {
+    // Use 100% of the space available for the input, but don't let the 10px
+    // padding on either side of the input to push it out of alignment.
+    width: calc(100% - 20px);
```

Comment:
Maybe `box-sizing` instead?

Reply:
![image](https://user-images.githubusercontent.com/1190097/53256625-8ddafc80-3685-11e9-81f4-eac827eb954f.png)

Nope - tried `border-box`, `padding-box`, and `content-box` and all 3 have very similar results.